src/main/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsView.java

```diff
@@ -0,0 +1,543 @@
+                });
+        journalFilesBox.itemsProperty().addListener((observable, oldvalue, newvalue) -> {
+            viewModel.journalFilesProperty().set(newvalue);
+        });
+        journalFilesBox.getSelectionModel().selectedItemProperty().addListener((observabe, oldvalue, newvalue) -> {
+            viewModel.changeActiveFile(newvalue);
+        });
+        viewModel.journalFilesProperty().addListener((observable, oldvalue, newvalue) -> {
+            journalFilesBox.itemsProperty().set(newvalue);
+        });
+        viewModel.abbreviationsProperty().addListener((observable, oldvalue, newvalue) -> {
+            journalAbbreviationsTable.setItems(newvalue);
+        });
+        journalAbbreviationsTable.itemsProperty().addListener((observable, oldvalue, newvalue) -> {
+            viewModel.abbreviationsProperty().set(newvalue);
+        });
+        isEditableAndRemovable.addListener((observable, oldvalue, newvalue) -> {
+            removeJournalAbbreviationsButton.setDisable(newvalue.booleanValue());
+        });
+    }
+
+    public void showAndWait() {
+        FXAlert journalAbbreviationsDialog = new FXAlert(AlertType.INFORMATION,
+                Localization.lang("Journal abbreviations"));
+        journalAbbreviationsDialog.setResizable(true);
+        journalAbbreviationsDialog.setDialogPane((DialogPane) this.getView());
+        ((Stage) this.getView().getScene().getWindow()).setMinHeight(400);
+        ((Stage) this.getView().getScene().getWindow()).setMinWidth(600);
+        journalAbbreviationsDialog.showAndWait();
+    }
+
+    @FXML
+    private void addNewFile() {
+        FileChooser chooser = new FileChooser();
+        chooser.getExtensionFilters().add(new ExtensionFilter("TXT files (*.txt)", "*.txt"));
+        File file = chooser.showSaveDialog(null);
+        if (file != null) {
+            try {
+                viewModel.addNewFile(file.getAbsolutePath());
+                journalFilesBox.getSelectionModel().selectLast();
+            } catch (JabRefException e) {
+                showErrorDialog(e);
+            }
+        }
+    }
+
+    @FXML
+    private void openFile() {
+        FileChooser chooser = new FileChooser();
+        chooser.getExtensionFilters().add(new ExtensionFilter("TXT files (*.txt)", "*.txt"));
+        File file = chooser.showOpenDialog(null);
+        if (file != null) {
+            try {
+                viewModel.openFile(file.getAbsolutePath());
+                journalFilesBox.getSelectionModel().selectLast();
+            } catch (JabRefException e) {
+                showErrorDialog(e);
+            }
+        }
+    }
+
+    @FXML
+    private void removeList() {
+        viewModel.removeCurrentList();
+    }
+
+    @FXML
+    private void fileChanged() {
+        viewModel.changeActiveFile(journalFilesBox.getSelectionModel().getSelectedItem());
+    }
+
+    @FXML
+    private void addAbbreviation() {
+        viewModel.abbreviationsNameProperty().set("Name");
```

Comment:
Can these two set-lines also be moved to `viewModel.addAbbreviation`? Also localize the strings.